### PR TITLE
sgt-back-end

### PR DESCRIPTION
**Pull request by nhuphham**

### CODE REVIEW

File: sgt-back-end/index.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  const sql = `
+  delete from "grades"
+  where "gradeId" = $1
+  returning *;
+  `;
+  db.query(sql, params)
+    .then(result => {
+      if (result.rowCount === 1) {
+        const message204 = { success: `gradeId ${id} was deleted.` };
+        return res.status(204).send(message204);
```
Comment (scbowler) [inline sgt-back-end/index.js:117]: With a status of `204` you can't send any data in the response. `204` means success with No Content. So here it is better to use `sendStatus`
```JAVASCRIPT
return res.sendStatus(204);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one update needed. Let me know if you have any questions.

File: sgt-back-end/index.js (hunk 1 above)
Comment (nhuphham) [reply sgt-back-end/index.js]: Screen Shot 2022-09-29 at 5 57 10 PM

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**